rootfs/scheduler/__init__.py

```diff
@@ -176,13 +177,14 @@ def _get_deploy_batches(self, steps, desired):
 
         return batches
 
-    def _update_application_service(self, namespace, name, app_type, port, routable=False):
+    def _update_application_service(self, namespace, name, app_type, port, routable=False, maintenance=False):  # noqa
```

Comment:
Could use https://github.com/deis/controller/blob/master/rootfs/api/models/__init__.py#L52-L94 if we want to go as far as naming it after a particular component - That's what `Domains` currently uses